packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -24,8 +24,12 @@ export type SliderSlots = {
    * The hidden input for the Slider.
    * This is the PRIMARY slot: all native properties specified directly on `<Slider>` will be applied to this slot,
    * except `className` and `style`, which remain on the root slot.
+   *
+   * Orient is a non standard attribute that allows for vertical orientation in Firefox
+   * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range#non_standard_attributes
+   * Webkit/blink support provided via css property
```

Comment:
This is confusing, or at least lacking in details: what css property provides support? What does a user of this component need to do with this information about css support? Also, what about chromium?

It might also be worthwhile mentioning that this property is handled internally based on Slider's `vertical` prop, and doesn't need to be set by the user.